_node-machine_to_Syncano-script/createAccount.js

```diff
@@ -0,0 +1,37 @@
+const Syncano = require('syncano');
+const fs = require('fs');
+let connection = Syncano({baseUrl: 'https://api.syncano.rocks'});
+const generateCiCredentials = require('./generateCiCredentials');
+
+var Account = connection.Account;
+var credentials = generateCiCredentials();
+let api_key;
+let instance_name;
+
+function account() {
+  return Account
+  .register(credentials)
+  .then((account) => {
+    connection = Syncano({accountKey: account.account_key, baseUrl: 'https://api.syncano.rocks'});
```

Comment:
`'https://api.syncano.rocks'` it should be taken from helpers, and you shouldn't override connection, instead you should set `accountKey`